participants/GammaCloud.md

```diff
@@ -0,0 +1,100 @@
+# GammaCloud
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+Our Project Name is GammaCloud. Our Team Slack Handles is `UK64776JJ`, and with Display name `RockYang`.
+
+## (1) Brief Project Description
+
+GammaCloud is a cloud file system baseed on Filecoin and IPFS.
+
+Any enterprise users and individual users can back up their data to GammaCloud. Usually users can only see the data uploaded by themselves. 
+
+We will also upload some open datasets to allow user to download publicly.
```

Comment:
it would be great if you can specify the initial datasets (realizing that they may change during the course of the competition)

Reply:
The initial datesets are linux system releases. Such as Ubuntu, CentOs... We will upload  series of versions for their ISO file.